src/states/adult/mod.rs

```diff
@@ -586,6 +585,15 @@ impl Base for Adult {
                 self.direct_msg_backlog.push((p2p_node, msg));
                 Ok(Transition::Stay)
             }
+            msg => {
```

Comment:
This is not ideal, we probably want to keep the exhaustive list in these matches because we need to look at them everytime a variant is added or changed.